apps/src/weblab/main.js

```diff
@@ -0,0 +1,19 @@
+var appMain = require('../appMain');
+var studioApp = require('../StudioApp').singleton;
+var WebLab = require('./WebLab');
+// var blocks = require('./blocks');
+var skins = require('../skins');
+// var levels = require('./levels');
```

Comment:
Delete comments?

Could make these require's into imports, but it might be worth leaving as is if all of our other `main.js` files look similar to this.